android/src/main/kotlin/com/onegini/mobile/sdk/flutter/facade/BiometricPromptFacadeImpl.kt

```diff
@@ -0,0 +1,44 @@
+package com.onegini.mobile.sdk.flutter.facade
+
+import javax.inject.Inject
+import javax.inject.Singleton
+import android.app.Activity
+import androidx.fragment.app.FragmentActivity
+import androidx.biometric.BiometricPrompt
+import androidx.core.content.ContextCompat
+import com.onegini.mobile.sdk.flutter.handlers.BiometricAuthenticationRequestHandler
+import com.onegini.mobile.sdk.flutter.pigeonPlugin.OWBiometricMessages
+
+@Singleton
+class BiometricPromptFacadeImpl @Inject constructor() : BiometricPromptFacade {
+
+  private lateinit var biometricPrompt: BiometricPrompt
+
+  override fun showPrompt(biometricMessages: OWBiometricMessages, activity: Activity, biometricRequestHandler: BiometricAuthenticationRequestHandler) {
+    val fragmentActivity = activity as FragmentActivity
+    biometricPrompt = BiometricPrompt(fragmentActivity, ContextCompat.getMainExecutor(fragmentActivity), object: BiometricPrompt.AuthenticationCallback() {
+      override fun onAuthenticationError(errorCode: Int, errString: CharSequence) {
+        biometricRequestHandler.CALLBACK?.onBiometricAuthenticationError(errorCode)
+      }
+
+      override fun onAuthenticationSucceeded(result: BiometricPrompt.AuthenticationResult) {
+        biometricRequestHandler.CALLBACK?.userAuthenticatedSuccessfully()
+      }
+
+      override fun onAuthenticationFailed() {
+      }
+    })
+
+    val promptInfo: BiometricPrompt.PromptInfo = BiometricPrompt.PromptInfo.Builder()
+      .setTitle(biometricMessages.title)
+      .setSubtitle(biometricMessages.subTitle)
+      .setNegativeButtonText(biometricMessages.negativeButtonText)
+      .setDescription(biometricMessages.description)
+      .build()
+    biometricPrompt.authenticate(promptInfo, biometricRequestHandler.CRYPTO_OBJECT!!)
```

Comment:
I think there should be something like this just in case crypto object is null for some reason. To avoid crashing the plugin
```kotlin
biometricRequestHandler.CRYPTO_OBJECT?.let {
   biometricPrompt.authenticate(promptInfo, it)
}
```

Reply:
Updated and also fixed below warning shown in BiometricPromptFacade class during the build i.e. renamed parameters in BiometricPromptFacade. 

warning : The corresponding parameter in the supertype 'BiometricPromptFacade' is named 'messages'. This may cause problems when calling this function with named arguments.